### DISCUSSION TITLE
Click on an icon to see all the selected options

### DISCUSSION THREAD

**Discussion by mohsin-j777** [brianvoe/slim-select#554] [Ideas]
**Describe the bug**
<img width="1432" alt="image" src="https://github.com/brianvoe/slim-select/assets/80242734/515ac201-da7a-45e3-9743-a3c7875bdc01">

As you can see the box here has a huge collection of selected items. What I want, is to display only 10 items  with `...` icon and have other selected options as hidden. So when I click on `...` icon, the container expands and show all the selected options. Like its doing in the picture.

Reason: I don't want to limit maxValuesShown: 10 and display the text `10 selected` .

Do you have any idea, how can I achieve this?

Thanks!

**Comment by brianvoe** [comment]
sorry i do not have an answer for this.
